.vscode/settings.json

```diff
@@ -1,26 +1,31 @@
 {
-    "eslint.validate": [
-        "javascript",
-        "javascriptreact",
-        "typescript",
-        "typescriptreact"
-    ],
-    "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true,
-        "source.fixAll.stylelint": true,
-    },
-    "editor.formatOnSave": true,
-    "typescript.format.enable": false,
-    "javascript.format.enable": false,
-    "cssVariables.lookupFiles": [
-        "src/styles/theme.scss",
-        "src/styles/themes/_light.scss", // include one of the theme file
-    ],
-    "prettier.disableLanguages": [
-        "javascript",
-        "javascriptreact",
-        "typescript",
-        "typescriptreact",
-        "scss"
-    ],
+  "eslint.validate": [
+    "javascript",
+    "javascriptreact",
+    "typescript",
+    "typescriptreact"
+  ],
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true,
+    "source.fixAll.stylelint": true
+  },
+  "editor.formatOnSave": true,
+  "typescript.format.enable": false,
+  "javascript.format.enable": false,
+  "cssVariables.lookupFiles": [
+    "src/styles/theme.scss",
+    "src/styles/themes/_light.scss" // include one of the theme file
+  ],
+  "prettier.disableLanguages": [
+    "javascript",
+    "javascriptreact",
+    "typescript",
+    "typescriptreact",
+    "scss"
+  ],
+  "i18n-ally.localesPaths": [
```

Comment:
Salam alykum Sameer,

JAK for your contribution!

The i18n Ally extension looks cool, thanks for including it! But can you please remove it from this PR and make a new one for it as it's out of scope?

Reply:
Walaikumsalaam.
Jzk, have updated in recent commit